admin/tool/policy/classes/api.php

```diff
@@ -331,6 +331,22 @@ public static function inactivate($policyid) {
         $DB->set_field('tool_policy', 'currentversionid', null, ['id' => $policyid]);
     }
 
+    /**
+     * Check if the user is a digital minor.
+     *
+     * @param int $dateofbirth
+     * @param string $country
+     * @return bool
+     */
+    public static function is_minor($dateofbirth, $country) {
```

Comment:
I believe it was agreed not to ask for the date of birth. There should be just "How old are you?" and "Where do you live?" questions.

Reply:
This change was made in discussion with the UX team. However, this can be further discussed and changed back to age if most of the people support that idea.